ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -94,7 +94,12 @@ octets=($(echo "${service_range}" | sed -e 's|/.*||' -e 's/\./ /g'))
 service_ip=$(echo "${octets[*]}" | sed 's/ /./g')
 
 # Determine appropriete subject alt names
-declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain} DNS:${master_name})
+declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain})
```

Comment:
I mean, this `cert_ip` was assign by `MASTER_IP` which was an env variable defined in task `Run create cert script on master`.

You can finally find out that `MASTER_IP` was assign by ansible fact 
`{{ hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | join (',IP:') }}` which include all IP address from host's network interfaces. 

The problem this that in multi-masters case, e.g master A (IP: 10.0.0.1) and master B (IP: 10.0.0.2), the `cert_ip` only include "10.0.0.1" if master A was `groups['masters'][0]`.